h/static/styles/partials-v2/_annotation-card.scss

```diff
@@ -54,3 +54,9 @@
   letter-spacing: 0.1px;
   border-left: 3px solid #d3d3d3;
 }
+
+.annotation-card__text {
+  img {
```

Comment:
This works, although there are a few other tags that are also styled inconsistently with the stream/client such as `<code>` and `<blockquote>`.

In the client, the `markdown-body` class which displays annotation markdown gets most of its styling by importing the `styled-text` mixin, which still exists in this repository in `partials/_styled-text.scss`.

What we could do is:

1. Extract the mixin from `partials/_styled-text.scss` into `mixins/_styled-text.scss` and add that to `mixins/_all.scss`
2. Include the mixin in the `annotation-card__text` class.

This will mean that all of the styling for annotation cards is in one mixin file and if we make changes to it, we can easily update the copy in the client repository as well.